src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStreamAsyncMemoryOverloads.Fixer.cs

```diff
@@ -30,12 +30,11 @@ namespace Microsoft.NetCore.Analyzers.Runtime
     /// </summary>
     public abstract class PreferStreamAsyncMemoryOverloadsFixer : CodeFixProvider
     {
+        private static readonly SyntaxAnnotation s_asMemorySymbolAnnotation = new("SymbolId", "System.MemoryExtensions");
```

Comment:
This is quite an implementation detail on Roslyn. I have an API proposal to expose it in cleaner way. https://github.com/dotnet/roslyn/issues/65041